### DISCUSSION TITLE
Found a little fix for .md file import in the actual @next v2.0.0-beta.4 branch

### DISCUSSION THREAD

**Discussion by Julien-cpsn** [quasarframework/quasar-ui-qmarkdown#364] [General]
Hi !

As I understood in the changelog, @hawkeye64 said that .md and .vmd files are no longer suported, so... it kind of breaks QMarkdown file import. Since it was my case, it broke my application (can't build). Saddly found no help on internet, but I did found this little temporary fix. Load the .md files as raw text. Does work fine for me :)

```
// To remove when the QMarkdown package is fixed
extendWebpack (cfg) {
  cfg.module.rules.push({
    test: /\.md$/,
    loader: 'raw-loader'
  })
},
```

Hope it'll be usefull for someone and that I did understood the changelog, otherwise sorry for this post ^^

Have a great day !

**Comment by hawkeye64** [comment]
@Julien-cpsn I'll look into adding that part back in. I need to find an alternative way for when devs are using app-vite instead of app-webpack.

**Comment by mickey-bimlive** [comment]
Using .md files as input to QMarkdown has been critical for my app. I'm currently upgrading the app from Quasar v1 to Quasar v2 with Vue 3, so I'm struggling with this issue. I have tried the above workaround, using the raw-loader. It does the rendering from .md-files, but when I do a change in the markdown in the .md file, the app (in dev mode) is not reflecting the change - only after a reload of the browser the update is rendered. That was different with Quasar v1. What is the Quasar v2/Vue 3 guidance from a QMarkdown perspective, if displaying markdown content from files is a critical requirement?

**Reply by Julien-cpsn** [reply]
I didn't succeed to replicate your bug, on my dev server updating a .md file updates the text aswell (maybe cause of hot reload?). There's probably solutions for that!

**Even though I don't have your problem, there are my suggestions to test :**

Load file like this instead of webpack rule
```
import ABOUT from 'raw-loader!../../ABOUT.md'
```

Use ref to force reactivity
```
import ABOUT from '../../ABOUT.md'
import { ref } from "vue";

...
setup() {
    return {
        ABOUT: ref(ABOUT),
    }
}
```

Both solutions!?

Find a way to make a reactive loader?

*Hope it helped you!*

-----

Personnaly, loading markdown from md files is mendatory and if this feature disapears I'll probably go on another library

Have a great day !

**Comment by mickey-bimlive** [comment]
Thanks, @Julien-cpsn - not sure why the hot reload after a change in the md file doesn't work in my case. Unsure whether the raw-loader that you suggested is the cause of it. I'll play with your further suggestions.

@hawkeye64: Agree with Julien that support of markdown from files is key.

**Comment by hawkeye64** [comment]
Working on it, but having some issues.

**Comment by hawkeye64** [comment]
You can now import *.md files (not *.vmd files) in v2.0.0-beta.6 release
https://github.com/quasarframework/quasar-ui-qmarkdown/releases/tag/v2.0.0-beta.6
